src/utils/apicalls.js

```diff
@@ -1,14 +1,23 @@
 import axios from 'axios';
 
 const newsApi = axios.create({
-    baseURL: 'https://be-nc-news-unk8.onrender.com/'
+    baseURL: 'https://be-nc-news-unk8.onrender.com'
 })
 
 export const fetchArticles = (page) => {
     return newsApi
     .get('/api/articles', {params: {
         p: page
     }})
+    .then((response) => {
```

Comment:
There's a nice opportunity to neaten up this code with by destructuring the body from the response - eg 
`.then(({ body }) => {`
Or even use some nested destructuring - eg `.then(({ body: { articles } }) => {`
If you choose to refactor this, make sure you adjust the way your components to reflect the different return value from this function